app/components/participant_status_tag_component.rb

```diff
@@ -16,26 +17,38 @@ def call
+
+  def status
+    @status ||= ::Participants::StatusAtSchool.new(induction_record:, has_mentees:, profile:).call
+  end
 
   def tag_attributes
-    return { text: "Withdrawn by provider", colour: "red" } if training_status_withdrawn?
-    return { text: "Eligible to start", colour: "green" } if eligible? && profile.single_profile?
-    return { text: "Eligible: Mentor at main school", colour: "green" } if eligible? && profile.primary_profile?
-    return { text: "Eligible: Mentor at additional school", colour: "green" } if ineligible? && mentor_with_duplicate_profile?
-
-    return { text: "Not eligible: No QTS", colour: "red" } if participant_has_no_qts?
-    return { text: "DfE checking eligibility", colour: "orange" } if profile.manual_check_needed?
-    return { text: "Not eligible: NQT+1", colour: "red" } if nqt_plus_one? && ineligible?
-    return { text: "Eligible to start: ERO", colour: "green" } if ineligible? && mentor_was_in_early_rollout? && on_fip?
-    return { text: "Eligible to start", colour: "green" } if ineligible? && mentor_was_in_early_rollout?
-    return { text: "Not eligible", colour: "red" } if ineligible?
-    return { text: "Contacted for information", colour: "grey" } if latest_email&.delivered?
-    return { text: "Check email address", colour: "grey" } if latest_email&.failed?
-
-    { text: "Contacting for information", colour: "grey" }
+    if FeatureFlag.active?(:cohortless_dashboard)
+      {
+        text: t(:header, scope: translation_scope),
+        colour: t(:colour, scope: translation_scope),
+      }
+    else
+      return { text: "Withdrawn by provider", colour: "red" } if training_status_withdrawn?
+      return { text: "Eligible to start", colour: "green" } if eligible? && profile.single_profile?
+      return { text: "Eligible: Mentor at main school", colour: "green" } if eligible? && profile.primary_profile?
+      return { text: "Eligible: Mentor at additional school", colour: "green" } if ineligible? && mentor_with_duplicate_profile?
+
+      return { text: "Not eligible: No QTS", colour: "red" } if participant_has_no_qts?
+      return { text: "DfE checking eligibility", colour: "orange" } if profile.manual_check_needed?
+      return { text: "Not eligible: NQT+1", colour: "red" } if nqt_plus_one? && ineligible?
+      return { text: "Eligible to start: ERO", colour: "green" } if ineligible? && mentor_was_in_early_rollout? && on_fip?
+      return { text: "Eligible to start", colour: "green" } if ineligible? && mentor_was_in_early_rollout?
+      return { text: "Not eligible", colour: "red" } if ineligible?
+      return { text: "Contacted for information", colour: "grey" } if latest_email&.delivered?
+      return { text: "Check email address", colour: "grey" } if latest_email&.failed?
+
+      { text: "Contacting for information", colour: "grey" }
+    end
   end
 
+  # Remove this code when we remove FeatureFlag.active?(:cohortless_dashboard) - start #
```

Comment:
I think we can remove this comment.

Reply:
This are comments I have put to facilitate the job to be done in the future when we definitely go for cohortless.
At that moment, the feature flag :cohortless_dashboard won't make sense anymore and we should remove it from the service.
If we do, the methods inside those comments will become unreachable, never called from the rest of the service, and would be good to remove them.
These comments are to facilitate the quick detection of those methods and wipe them!